browser-test/src/admin_program_translation.test.ts

```diff
@@ -148,6 +152,123 @@ describe('Admin can manage translations', () => {
+    await adminProgramImage.setImageDescriptionAndSubmit(
+      'Fake image description',
+    )
+
+    await adminPrograms.gotoDraftProgramManageTranslationsPage(programName)
+    await adminTranslations.selectLanguage('Spanish')
+    await adminTranslations.expectProgramImageDescriptionTranslation('')
+
+    // Add a Spanish translation
+    await adminPrograms.gotoDraftProgramManageTranslationsPage(programName)
+    await adminTranslations.selectLanguage('Spanish')
+    await adminTranslations.editProgramTranslations({
+      name: 'Spanish name',
+      description: 'Spanish description',
+      statuses: [],
+    })
+    await adminTranslations.editProgramImageDescription(
+      'Spanish image description',
+    )
+
+    await adminPrograms.gotoDraftProgramManageTranslationsPage(programName)
+    await adminTranslations.selectLanguage('Spanish')
+    await adminTranslations.expectProgramImageDescriptionTranslation(
+      'Spanish image description',
+    )
+    await validateScreenshot(page, 'program-translation-with-image-description')
+
+    // Verify other translations are still not filled in
+    await adminPrograms.gotoDraftProgramManageTranslationsPage(programName)
+    await adminTranslations.selectLanguage('Tagalog')
+    await adminTranslations.expectProgramImageDescriptionTranslation('')
+
+    await disableFeatureFlag(page, 'program_card_images')
+  })
+
+  it('editing summary image description does not clobber translations', async () => {
+    const {page, adminPrograms, adminProgramImage, adminTranslations} = ctx
+    await loginAsAdmin(page)
+    await enableFeatureFlag(page, 'program_card_images')
+
+    const programName = 'Program with summary image description'
+    await adminPrograms.addProgram(programName)
+    await adminPrograms.goToProgramImagePage(programName)
+    await adminProgramImage.setImageDescriptionAndSubmit(
+      'Fake image description',
+    )
+
+    await adminPrograms.gotoDraftProgramManageTranslationsPage(programName)
+    await adminTranslations.selectLanguage('Spanish')
+    await adminTranslations.editProgramTranslations({
+      name: 'Spanish name',
+      description: 'Spanish description',
+      statuses: [],
+    })
+    await adminTranslations.editProgramImageDescription(
+      'Spanish image description',
+    )
+
+    // Update the original description
+    await adminPrograms.goToProgramImagePage(programName)
+    await adminProgramImage.setImageDescriptionAndSubmit(
+      'New image description',
+    )
+
+    // Verify the Spanish translations are still there
+    await adminPrograms.gotoDraftProgramManageTranslationsPage(programName)
+    await adminTranslations.selectLanguage('Spanish')
+    await adminTranslations.expectProgramImageDescriptionTranslation(
+      'Spanish image description',
+    )
+
+    await disableFeatureFlag(page, 'program_card_images')
+  })
+
+  it('deleting summary image description deletes all translations', async () => {
```

Comment:
question (non-blocking): Is this testing that the translations are deleted because the translation field will only show if they are present?

Reply:
Yes exactly! I added a bit more to the comment near the end of the method -- let me know if it makes sense or if there's a better way to explain.

Reply:
Thanks perfect. Thank you!